deploy/csidriver.yaml

```diff
@@ -3,5 +3,11 @@ kind: CSIDriver
 metadata:
   name: cstor.csi.openebs.io
 spec:
-  attachRequired: true
+  # Supports persistent and ephemeral inline volumes.
```

Comment:
what is the need of this file?

Reply:
The CSIDriver Kubernetes API object serves two purposes:

-  Simplify driver discovery : 
If a CSI driver creates a CSIDriver object, Kubernetes users can easily discover the CSI Drivers installed on their cluster (simply by issuing kubectl get CSIDriver)

- Customizing Kubernetes behavior:
Kubernetes has a default set of behaviors when dealing with CSI Drivers (for example, it calls the Attach/Detach operations by default). This object allows CSI drivers to specify how Kubernetes should interact with it.

more info https://kubernetes-csi.github.io/docs/csi-driver-object.html